addons/purchase/security/ir.model.access.csv

```diff
@@ -24,7 +24,6 @@ access_account_tax_purchase_user_manager,account.tax,account.model_account_tax,g
 access_product_product_purchase_user,product.product.purchase.user,product.model_product_product,group_purchase_user,1,0,0,0
 access_product_product_purchase_manager,product.product purchase_manager,product.model_product_product,purchase.group_purchase_manager,1,1,1,1
 access_product_template_purchase_user,product.template purchase_user,product.model_product_template,group_purchase_user,1,0,0,0
-access_product_template_purchase_manager,product.template purchase_manager,product.model_product_template,group_purchase_manager,1,0,0,0

```

Comment:
It's duplicated from https://github.com/odoo/odoo/blob/1d34e269dac2e0cafd85ad91a6240569b5b7282c/addons/purchase/security/ir.model.access.csv#L44